sim/rogue/items_sets_pve.go

```diff
@@ -181,7 +183,7 @@ var ItemSetNightSlayerBattlearmor = core.NewItemSet(core.ItemSet{
 		},
 		// Vanish now reduces all Magic damage you take by 50% for its duration, but it no longer grants Stealth or breaks movement impairing effects.  - 457437
 		4: func(agent core.Agent) {
-			// Immplemented in Vanish.go
+			// Implemented in Vanish.go
 		},
 		// Your finishing moves have a 20% chance per combo point to make you take 50% less Physical damage from the next melee attack that hits you within 10 sec.
```

Comment:
I just realized looking at the code for this set bonus that it should use explicitly a physical school damage taken mod. So it would be `Pseudostats.SchoolDamageTakenMultiplier[stats.SchoolIndexPhysical]`

Reply:
fixed!